pkgs/development/compilers/rust/default.nix

```diff
@@ -18,6 +18,7 @@
 , CoreFoundation, Security, SystemConfiguration
 , pkgsBuildTarget, pkgsBuildBuild
 , makeRustPlatform
+, fast_cross ? ((stdenv.buildPlatform == stdenv.hostPlatform) && (stdenv.hostPlatform != stdenv.targetPlatform))
```

Comment:
I'm not sure this should be overrideable, since it presumably wouldn't work in any other case, and shouldn't ever be the wrong thing to do in this case.